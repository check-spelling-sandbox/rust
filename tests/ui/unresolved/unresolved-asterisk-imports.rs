use nonexistent_crate::*; //~ ERROR unresolved import `nonexistent_crate
use std as foo;

fn main() {}
