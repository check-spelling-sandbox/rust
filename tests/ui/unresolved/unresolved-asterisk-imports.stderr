error[E0432]: unresolved import `nonexistent_crate`
  --> $DIR/unresolved-asterisk-imports.rs:1:5
   |
LL | use nonexistent_crate::*;
   |     ^^^^^^^^^^^^^^^^^ maybe a missing crate `nonexistent_crate`?
   |
   = help: consider adding `extern crate nonexistent_crate` to use the `nonexistent_crate` crate

error: aborting due to previous error

For more information about this error, try `rustc --explain E0432`.
